qutebrowser/config/configtypes.py

```diff
@@ -806,19 +819,21 @@ def validate(self, value):
                 raise configexc.ValidationError(value, "may not be empty!")
         value = os.path.expanduser(value)
         try:
-            if not os.path.isfile(value):
-                raise configexc.ValidationError(value, "must be a valid file!")
             if not os.path.isabs(value):
+                cfgdir = standarddir.config()
+                if cfgdir is not None and os.path.isfile(
+                        os.path.join(cfgdir, value)):
+                    return
                 raise configexc.ValidationError(
-                    value, "must be an absolute path!")
+                    value, "must be an absolute path when not using a config "
+                    "directory!")
+            elif not os.path.isfile(value):
+                raise configexc.ValidationError(
+                    value, "must be a valid path relative to the config "
+                    "directory!")
```

Comment:
This would be raised if the user gives you an *absolute* path to something else than an existing file, so I think this should just say `"must be a valid file!"` or so.